### ISSUE TITLE
Obtain the editing status of the document and save the document through the DocEditer API

### ISSUE THREAD

**Issue by 277801235** [ONLYOFFICE/DocumentServer#1538]
I expect that when the browser window is closed, if there is a modification for saving, it will prompt whether to keep the uncommitted modification, and the document can be forced to be saved through the API.

**Comment by ShockwaveNN** [comment]
> it will prompt whether to keep the uncommitted modification

You get this modification by default (at least using integrated test example)

![image](https://user-images.githubusercontent.com/668524/144802457-c95e5c72-c6a5-4e6c-a28e-0b338dbc7319.png)

1. I've created a new file
2. Input a little bit of text and immediately pressed closed tab, until autosave worked (as far as I remember it works every 2 seconds)

**Comment by 277801235** [comment]
Thank you for your reply

It runs very well in the browser. But when I integrate it into the Electron window, it doesn't work. So I want to be able to handle the onbeforeunload event by myself

**Comment by ShockwaveNN** [comment]
> But when I integrate it into the Electron window

I don't think we ever tested integrated to electron window and I don't think I've enough knowledge in that base to figure something out

So any help - welcomed here
